### ISSUE TITLE
Test ConvertGDb genbank in Windows

### ISSUE THREAD

**Issue by ssnn-airr** [immcantation/changeo#163]
**[Original report](https://bitbucket.org/kleinstein/changeo/issues/163) by ssnn (Bitbucket: [ssnn](https://bitbucket.org/ssnn), GitHub: [ssnn-airr](https://github.com/ssnn-airr)).**

----------------------------------------

There is some glitch with ConvertDb genbank and the wrapping of tbl2asn. Need to test this function in Windows.

Sometimes tbl2asn fails to recognize the input fasta \(see also #137\):

> There is a known issue with the `--asn` argument. In some environments, for reasons that are presently unknown, **tbl2asn** may fail to recongizing the input fasta file and report an error stating **Unable to read any FASTA records**. Running **tbl2asn** manually should resolve the issue.

Sometimes some sequences are missing in the output \(see also #162\):

> It is recommended that the number of sequences in the output `sqn` file be verified against the number of sequences in the input `tab` or `tsv` file. From the command line, this can be achieved via `grep -c iupacna *.sqn`. This step is not necessary if running **tbl2asn** outside [ConvertDb](https://changeo.readthedocs.io/en/latest/tools/ConvertDb.html#convertdb).

‌

**Comment by ssnn-airr** [comment]
**Original comment by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

For the missing sequences, is there a discrepancy between the number of records in the output and the PASS/FAIL counts shown in the console log? Filtering of incomplete sequences is intentional.

**Comment by ssnn-airr** [comment]
**Original comment by Julian Zhou (Bitbucket: [jqz](https://bitbucket.org/jqz), GitHub: [julianqz](https://github.com/julianqz)).**

----------------------------------------

The console log says all passed. But when you actually look in the `sqn`, not all sequences are there. Problem goes away when running `tbl2asn` by itself outside `ConvertDb`. Also, since I ran `ConvertDb` a few times with `--asn` on as I was doing resubmission multiple times, I eventually noticed that the number of sequences in `sqn` varied slightly from submission to submission, suggesting that the number of sequences being omitted each time was different.
